static/js/publisher/release/revisionsList.js

```diff
@@ -11,35 +11,46 @@ export default class RevisionsList extends Component {
     this.props.selectRevision(revision);
   }
 
-  renderRows(revisions) {
+  renderRows(revisions, showCheckboxes) {
     return revisions.map(revision => {
-      const uploadDate = new Date(revision.created_at);
+      const revisionDate = revision.release
+        ? new Date(revision.release.when)
+        : new Date(revision.created_at);
       const isSelected = this.props.selectedRevisions.includes(
         revision.revision
       );
+
+      // disable revisions from the same architecture that already selected
+      // but only if checkboxes are visible (not in channel history)
       const isDisabled =
+        showCheckboxes &&
```

Comment:
is it worth using a different variable name that's more descriptive like "inChannelHistory" that both the checkboxes and this can use?